### PR TITLE
Add common processes section to readme - db schema

### PR DESCRIPTION
**Pull request by JohnFarrellDev**
## Link to JIRA ticket

No ticket

### *What is the problem we're trying to solve*

Document common problems so that the whole team knows how to solve them, reduce bus factor.

### *What changes have we introduced*

Documented how we update the database/schema.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,12 @@ It is a part of the Social Care system (see [Social Care System Architecture](ht
     - [Dockerised dependencies](#dockerised-dependencies)
     - [Installation](#installation)
   - [Contributing](#contributing)
-  - [Usage](#usage)
+  - [Common Processes](#common-processes)
     - [Running the application](#running-the-application)
     - [Running the tests](#running-the-tests)
+      - [Using the terminal](#using-the-terminal)
```
Comment (Renny4Real) [inline README.md:19]: 🙌🏾

File: README.md [hunk 2]
```diff
@@ -118,6 +121,21 @@ $ make start-test-dbs
 
 This will allow you to run the tests as normal in your IDE.
 
+### Updating the Schema
+
+- In the database repository update the schema.sql file
```
Comment (Renny4Real) [inline README.md:126]: Should we add a link to the schema file? 🤔

File: README.md [hunk 3]
```diff
@@ -118,6 +121,21 @@ $ make start-test-dbs
...
+- In SocialCareCaseViewerAPI/Infrastructure either update an existing class or create a new class for the schema changes
+- To test locally run `make restart-db`
+- To deploy changes to AWS:
+    - Go to AWS account (staging or prod)
+    - Go to Systems Manager
+    - Go to Session Manager
+    - Choose `RDS jump box-Platform APIs (new)` and click `Start Session`, this allows us to have a CLI into the instance hosting our database
+    - Connect to PostgreSQL `psql --host=social-care-db-production.c1rcth1f5uzc.eu-west-2.rds.amazonaws.com --port=5600 --username=<username> --password=<password> --dbname=social_care`
```
Comment (Renny4Real) [inline README.md:134]: Would it be a risk if we show the hostname publicly? 🤔

**Review by Renny4Real — COMMENTED** [review]
Looks good! Just a couple of comments 👍🏾

File: README.md (hunk 3 above)
Comment (JohnFarrellDev) [reply README.md]: It should be fine but I have amended it. No nasty hackers please

[hackerman](https://i.kym-cdn.com/entries/icons/original/000/021/807/ig9OoyenpxqdCQyABmOQBZDI0duHk2QZZmWg2Hxd4ro.jpg)

**Pull request merged**